src/NuGet.Core/NuGet.PackageManagement/Utility/PackagesConfigLockFileUtility.cs

```diff
@@ -189,6 +271,46 @@ private static void RemoveUninstalledPackages(PackagesLockFile lockFile, IEnumer
             }
         }
 
+        public static PackagesLockFile FromPackagesConfigFile(
+            string pcFile,
+            NuGetFramework projectTfm,
+            string packagesFolderPath,
+            CancellationToken token)
+        {
+            if (pcFile == null) throw new ArgumentNullException(nameof(pcFile));
```

Comment:
This will cause the 5 line argument validation to become 20 lines. In general I agree we should use use blocks and have them on new lines, but even in this case?

Reply:
Yes